### PR TITLE
L3-67 add the ability to turn off the aws integration for local development environments

### PR DESCRIPTION
**Pull request by mgwozdz-unicon**
## Description
I added relatively simple steps that can be taken to disable the AWS integration for local development environments.

### Motivation and Context
This solves the issue of developers needing valid AWS credentials and a valid SQS queue setup to be able to run the middleware locally (or else having to comment out large chunks of code).

### Fixes
[L3-67](https://lumenlearning.atlassian.net/browse/L3-67)

## How Has This Been Tested?
1. I commented out all of the `cloud.aws` properties in the properties file to remove my AWS credentials from my local setup.
2. I attempted to run `mvn clean install spring-boot:run` to start the local dev environment, and noticed the failure.
3. I followed the steps in the new README section titled "Turning off AWS Integration for Local Development" (Ran`export spring_profiles_active=no-aws` to set env var and uncommented `spring.autoconfigure.exclude` property).
4. I ran `mvn clean install spring-boot:run` to start the local dev environment and noticed that there was no longer build output regarding AWS. I confirmed that the LTI 1.3 standard launch flow still functioned in Canvas.
5. I followed the steps in the new README section titled "Turning on AWS Integration for Local Development" (Ran `unset spring_profiles_active` to remove the env var, commented out the `spring.autoconfigure.exclude` property, and uncommented the `cloud.aws` properties in the properties file).
6. I ran `mvn clean install spring-boot:run` to start the local dev environment and noticed that the AWS output has returned to the build logs and that the tomcat started successfully.
7. I used the "Sample SQS Message for AGS/Grade Passback" section of the README file to post a valid message to the SQS queue and confirmed that grade passback still functioned.

---

## Checklist
- [x] I have reviewed the AC for this feature and my changes meet all requirements
- [x] (For UI changes) I have considered the accessibility of this feature (see https://www.a11yproject.com/checklist/ for a high-level checklist)
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have requested a review from at least one other dev

### CODE REVIEW

File: src/main/java/net/unicon/lti/service/sqs/impl/SQSMessageReceiver.java [hunk 1]
```diff
@@ -31,6 +32,7 @@
 import java.util.Locale;
 
 @Slf4j
+@Profile("!no-aws")
```
Comment (smith750) [inline src/main/java/net/unicon/lti/service/sqs/impl/SQSMessageReceiver.java:35]: This is nice!

File: README.md [hunk 2]
```diff
@@ -138,6 +138,33 @@ Setup SQS for AGS Grade Pass Back
 2. The URI for a SQS queue should be filled in for either `lti13.grade-passback-queue` or as `LTI13_GRADE_PASSBACK_QUEUE` environment variable.
 3. The region of the SQS queue should be filled in for either `cloud.aws.region.static` in the application.properties file or `AWS_REGION` as an environment variable.
 
+Turning off AWS Integration for Local Development
+-------------------------------------------------
+Prior to running `mvn clean install spring-boot:run`, do the following:
+1. `export spring_profiles_active=no-aws`
+2. Ensure that the following property is present and uncommented in the application.properties file:
```
Comment (leannethng) [inline README.md:145]: Should we mention here and below, that if we are If we are creating a local `application-local.properties` file, these properties will need to be updated there instead?

**Review by leannethng — APPROVED** [review]
This is awesome Mary! I just tested it and it works great! 

Thanks as always for your amazing documentation, it is so helpful to be able to test everything out! I left one small comment as I have an `application-local.properties` file at the root that is gitignored to store some of the custom properties so I needed to edit that file instead when testing this. I figure thats likely what we'd want folks to do when running the app locally to avoid accidentally pushing creds, but there may be another solution!

Thanks so much!

File: README.md (hunk 2 above)
Comment (mgwozdz-unicon) [reply README.md]: Good point - I've updated it to reference both files.

**Pull request merged**